### PR TITLE
add redir requirement to docs

### PR DESCRIPTION
**Pull request by vdloo**
extra dependency needed after https://github.com/ByteInternet/hypernode-vagrant/pull/182

### CODE REVIEW

**Comment by andreas-bytenl** [comment]
Is this because we added `host_ip: "0.0.0.0"`? Is that part needed? I know I personally wouldn't want my Vagrant boxes to listen on all interfaces of my host machine.

**Comment by vdloo** [comment]
It is because [previously](https://github.com/ByteInternet/hypernode-vagrant/blob/124f91a0d043ee32a11a5c3585500542023607ad/Vagrantfile#L79) no portforwarding was done for the lxc provider at all. Now the default behavior is to forward 80 and 3306, but yes good point I can imagine that binding on all ipv4 interfaces might not be desirable. But to disable it you could add `default_ports: false` to your local.xml to [disable it like before](https://github.com/pauluse/hypernode-vagrant/blob/4508967d07676cc6b72ac15eb367c5d20992350d/Vagrantfile#L108). I'll add that to the docs.

**Comment by vdloo** [comment]
by the way, this was the default behavior before for virtualbox already:

> host_ip (string) - The IP on the host you want to bind the forwarded port to. If not specified, it will be bound to every IP. By default, this is empty.

https://www.vagrantup.com/docs/networking/forwarded_ports.html

File: Documentation/archlinux_setup.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ If you encounter any issues relating to `vagrant-lxc` and not `hypernode-vagrant
 pacman -S rsync dnsmasq git lxc vagrant --needed --noconfirm
 ```
 
+You will also need to install [redir](https://linux.die.net/man/1/redir). Alternatively you could disable [port forwarding](https://www.vagrantup.com/docs/networking/forwarded_ports.html) by adding `default_ports: false` to your `local.xml`.
```
[the remark this answers is not in the document]
Comment (vdloo) [reply Documentation/archlinux_setup.md]: :grimacing:

**Pull request merged**